package.json

```diff
@@ -92,6 +92,7 @@
     "eslint-plugin-json-files": "^2.2.0",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "expect-type": "^0.17.3",
```

Comment:
If you rebase you'll get this in (and the eslint improvements)

Reply:
yeah no worries will rebase a bit later.